common/shlibs

```diff
@@ -49,6 +49,35 @@ libcidn.so.1 glibc-2.32_1
 libmvec.so.1 glibc-2.32_1
 libcddgmp.so.0 cddlib-0.94m_1
 libcdd.so.0 cddlib-0.94m_1
+libc.so.6 glibc-2.34_1
```

Comment:
This commit has been applied wrong, for some reason. These entries should be replaced, rather than added to the list (my commit in the original PR does this).

Reply:
> This commit has been applied wrong, for some reason. These entries should be replaced, rather than added to the list (my commit in the original PR does this correctly).

Argh, I think git might've mangled it.